ui/pages/confirmations/components/confirm/row/dataTree.tsx

```diff
@@ -20,49 +26,117 @@ export type TreeData = {
   type: string;
 };
 
+const FIELD = {
```

Comment:
Could this be an enum instead since we're in a `.tsx`, such as `Field.Amount` etc?

Reply:
👍🏼  Updated [78a1f99](https://github.com/MetaMask/metamask-extension/pull/26107/commits/78a1f99ad8e8486f08f66eac72b6e2bec52ec706)